### ISSUE TITLE
Drilling up via breadcrumbs fires off redraw event for every level traversed

### ISSUE THREAD

**Issue by lani-po** [highcharts/highcharts#21886]
#### Expected behaviour
Selecting any breadcrumb should cause the chart to redraw only once.

#### Actual behaviour
If I am X amount of levels deep in a drilldown, selecting the first breadcrumb to view the top level chart should not cause it redraw X amount of times.

#### Live demo with steps to reproduce
https://jsfiddle.net/n8yjwhLg/
1. Click on "Chrome" slice
2. Click on "v97.0" slice
3. Clear console
4. Click on "Browsers" breadcrumb
5. Notice via console that the chart redraws twice, causing the animation to jump a bit

#### Product version
Latest as of reporting

#### Affected browser(s)
All

**Comment by hubertkozik** [comment]
Hi @lani-po! To be honest, I don't see it as a big problem due to the correct animations of the chart.

I made the drilldown and drillup animations longer: https://jsfiddle.net/BlackLabel/f18bpcgv/
If I go to `Chrome`, then to `v97` and then click `Browsers` from breadcrumbs you can see two correct animations and those two redraws are correct in my opinion.

I don't think it's a bug, can I ask for your opinion @TorsteinHonsi?

**Comment by TorsteinHonsi** [comment]
I'm inclined to say it's a bug. As the OP says, the animation is a little bit jaggy, presumably because we start a new animation when the previous one is just started. Second, it is also not optimal in terms of performance to run expensive calculations and rendering that is redundant. 

I don't think it's very crucial though. It does work, albeit with a little jump. But if there's a simple solution to it, it could be worth checking out.

**Comment by lani-po** [comment]
@hubertkozik and @TorsteinHonsi I appreciate your prompt feedback! I've been using Highcharts for many years and have promoted it at every company I've worked at because I love its versatility and ease of use. I've always been a bit of a stickler for making sure I don't make the charts redraw unnecessarily whenever I implement one into our apps because I am a fan of the built-in smooth animations and optimal performance. This particular case stood out to me because I thought I was doing something wrong with how/when I was calling redraws manually.

As far as it being an actual bug, I agree with @TorsteinHonsi that it's not suuuper crucial, so I wouldn't blame you guys for flagging this as low priority. I'm more so concerned if a chart has drilldown levels even more than what I provided in the example. If I have a chart that's 5 levels deep, then does it make sense for the chart to redraw 5 times when going back to the top level? I'd be more okay with it if the animations transitioned smoothly to the next level level as it drills back up to emulate some sort of, uhh.. ascension? 😆 But as TorsteinHonsi said, the animations kind of just jarringly interrupt each other. 

But again, if it's more effort that it's worth then I'll just deal with the jumpiness for now :)
